DAS/validator.py

```diff
@@ -66,6 +134,22 @@ def __init__(self, ID, amIproposer, logger, shape, rows, columns):
         self.statsRxInSlot = 0
         self.statsRxPerSlot = []
 
+        # Set uplink bandwidth. In segments (~560 bytes) per timestep (50ms?)
+        # 1 Mbps ~= 1e6 / 20 / 8 / 560 ~= 11
+        # TODO: this should be a parameter
+        self.bwUplink = 110 if not self.amIproposer else 2200 # approx. 10Mbps and 200Mbps
+
+        self.repairOnTheFly = True
+        self.sendLineUntil = (self.shape.blockSize + 1) // 2 # stop sending on a p2p link if at least this amount of samples passed
+        self.perNeighborQueue = True # queue incoming messages to outgoing connections on arrival (as typical GossipSub impl)
+        self.shuffleQueues = True # shuffle the order of picking from active queues of a sender node
+        self.perNodeQueue = False # keep a global queue of incoming messages for later sequential dispatch
+        self.shuffleLines = True # shuffle the order of rows/columns in each iteration while trying to send
+        self.shuffleNeighbors = True # shuffle the order of neighbors when sending the same segment to each neighbor
+        self.dumbRandomScheduler = False # dumb random scheduler
+        self.segmentShuffleScheduler = True # send each segment that's worth sending once in shuffled order, then repeat
+        self.segmentShuffleSchedulerPersist = True # Persist scheduler state between timesteps
```

Comment:
Is there any realistic case in which we want this not to persist? 
Just curiosity, is good to have it as a parameter as well.

Reply:
if you persist between steps, you don't work with fresh information. On the practical side, persist seems to be a better model.